### PR TITLE
Add env variable to switch which site deploys in bodiless

### PR DESCRIPTION
**Pull request by konst8**
Testing cases

Prereq: npm run setup

1) No env variable set:
- npm run start -- expected Test Site in Edit mode -- PASS
- npm run build & npm run serve -- expected Test Site in Static mode -- PASS

2) env variable set to test-site
- export APP_SITE_NAME=test-site 
- npm run start -- expected Test Site in Edit mode -- PASS
- npm run build & npm run serve -- expected Test Site in Static mode -- PASS

3) env variable set to __cxstarter__
- export APP_SITE_NAME=__cxstarter__
- npm run start -- expected CanvasX Site in Edit mode -- PASS
- npm run build & npm run serve -- expected CanvasX Site in Static mode -- PASS

4) Verified test site is still coming up:
- http://pr-1491-vbuvbay-jvaff4bu65vgm.us-2.platformsh.site/ -- PASS
- http://edit.pr-1491-vbuvbay-jvaff4bu65vgm.us-2.platformsh.site/ -- FAIL

### CODE REVIEW

File: .platform.app.yaml [hunk 1]
```diff
@@ -100,3 +100,4 @@ variables:
     env:
         APP_VOLUME: '/app/volume'
         CHOKIDAR_USEPOLLING: true
+        APP_SITE_NAME: test-site
```
Comment (hvanyo) [inline .platform.app.yaml:103]: Do these need to change:  https://github.com/johnsonandjohnson/Bodiless-JS/blob/main/.platform.app.yaml#L86-L93

Comment (hvanyo) [reply .platform.app.yaml:103]: looking at how other site was done -- don't think so.

File: sites/test-site/edit/ecosystem.config.js [hunk 2]
```diff
@@ -1,18 +1,27 @@
+const fs = require('fs');
```
Comment (wodenx) [inline sites/test-site/edit/ecosystem.config.js:1]: I think if we are going to be choosing the site based on a variable, then we should move this direcotry up to the root as it is in canvasx.

**Review by Asemirski — APPROVED** [review]

**Comment by hvanyo** [comment]
@konst8  can this be can be closed and as you did this in another pr targeted to upgrade branch???

**Comment by hvanyo** [comment]
Closing as this not relevant anymore. Please reopen if this is still actual.

**Pull request closed without merging**